source/Halibut.Tests/Support/TestAttributes/LatestClientAndLatestServiceTestCasesAttribute.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using Halibut.Tests.Support.TestCases;
+using NUnit.Framework;
+
+namespace Halibut.Tests.Support.TestAttributes
+{
+    /// <summary>
+    ///     Holds all the standard test cases for testing a latest client with a latest service.
+    ///     In this case latest means the code as is, rather than some previously built version of halibut.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = true, Inherited = false)]
+    public class LatestClientAndLatestServiceTestCasesAttribute : TestCaseSourceAttribute
+    {
+        public LatestClientAndLatestServiceTestCasesAttribute(bool testWebSocket = true, bool testNetworkConditions = true) :
```

Comment:
nit: I wonder if network conditions could be an enum for description e.g.
```
TestNetworkCondtions : Network.All
TestNetworkCondtions : Network.PerfectOnly
```

maybe the bool reads better actually...

Reply:
Yeah I like the bool tbh - we're either testing network conditions, or the network is perfect. Either/Or, nice and simple